vaultwarden.json

```
@@ -0,0 +1,41 @@
+{
+    "Vaultwarden": {
+        "description": "Alternative implementation of the Bitwarden password manager server API written in Rust and on Alpine (speed, size). <p>Based on the following docker image: <a href='https://hub.docker.com/r/vaultwarden/server' target='_blank'>https://hub.docker.com/r/vaultwarden/server</a></p>.",
```

Add architecture compatibility:
```suggestion
        "description": "Alternative implementation of the Bitwarden password manager server API written in Rust and on Alpine (speed, size). <p>Based on the following docker image: <a href='https://hub.docker.com/r/vaultwarden/server' target='_blank'>https://hub.docker.com/r/vaultwarden/server</a> compatible with amd64 and arm64 architectures.</p>",
```